CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Added
+## Added
```

Comment:
just FYI we will want this to remain as a level 3 header `###`. it can be helpful to do a final review of the files changed view to just quickly run through and check for these little things.

https://github.com/flexcompute/tidy3d/pull/2542/files